### DISCUSSION TITLE
是否经常使用JavDB作为爬取目标?

### DISCUSSION THREAD

**Discussion by glyh** [Yuukiy/JavSP#469] [投票 | Polls]
目前，JavDB 的集成已损坏，并且维护它需要花费大量精力。鉴于我们用于维护该项目的资源有限，我想进行一次投票，以决定是否保留集成还是将其移除。

**Comment by 1zhouzhou** [comment]
给一些建议：如果不保留db，虽然问题不大，一些片的女优名这一项的获取会有问题，但不是大问题；db在这方面是主流中文站点里的佼佼者。但做的最好的还是小日子的sougouwiki和av-wiki，一些冷门片，小厂牌片也都非常全。sougouwiki更是将很多精选集片（比如FIV-xxx）以及合拍片（比如vrtm-xxx），的出处也列举出来，大多都给出了全部女优名和来自哪些单体番号，很方便。可惜sougouwiki不支持番号搜片，技术上它像一个简陋的excel表格。av-wiki的问题是偏科严重，小厂牌片极其强大，其他的就很一般，vrtm这种出到600部的系列都没有收录，属于一个还在成长的网站。打开话匣子，我顺便给一些个人评价，给项目维护的兄弟们一点小参考。
个人因为一些原因，会非常频繁使用javbus，JavDB，javlab，等等查询资料；就资料齐全度上来说，各有优劣；其实各家通病都集中在单片女优名缺失和错误上，这一点不难理解，日本片商出于一些奇怪的原因，很多片不提供女优名，然后女优因为移籍换名字，导致各种混乱，不过对刮削技术上不是问题，我看到新版sp有lib文件合并了女优名，这个点赞。
javbus基本不怎么维护片资料，抓取了片商的资料就不咋管了，管理员极为在意的是内部论坛；其远古资料还比较齐全，只是他们的资料也都是爬出来的，所以冷门资料不算多。
DB在资料完整度做得比bus好多了，单片女优名95%完整，很少看到女优那一项na，其远古资料相比bus各有长短，里番和欧美数据也是他的特色。
lab的问题是深恶痛绝的反爬机制，而且因为创建早的缘故，所以网站技术也很垃圾。个人感觉他家的资料比bus和db都要少，历史遗留问题一大堆，各方面其实都挺差的，我已经基本不用了。题外话，去年刚出的www.jav.land跟lab在界面上都很像，像是lab的3.0加强版。可惜就在上周已疑似跑路。
airav最大也是唯一的优势是全面汉化，连女优名都汉化了。
jav321的特点是图片和文本，提供了大图片和简介。
总结一下，如果项目维护的大大们能把sougouwiki的机制搞清楚，最终能实现通过番号刮削女优名，那db完全可以彻底无视掉。这方面它家是貌似是天花板。
